ftw/file/upgrades/20150504090723_enable_tiny_mce_image_upload/tinymce.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object>
+ <resourcetypes>
+  <imageobjects purge="True">
+    <element value="File"/>
```

Comment:
I'm not sure whether ftw.file should configure tinymce (globally) to only use `File` as image.
It seems to me quite invasive: we don't know whether the deployment also allows default `Image`s and which type should be used.
@maethu what do you think?